api/specs/webserver/openapi.yaml

```diff
@@ -168,6 +168,7 @@ paths:
   /projects/{project_id}/state:
     $ref: "./openapi-projects.yaml#/paths/~1projects~1{project_id}~1state"
 
+  # FIXME: /projects/{project_id}:export dose not work
   /projects/{project_id}/export:
```

Comment:
minor: this should be a POST function.
```
Custom methods should use HTTP POST verb since it has the most flexible semantics, except for methods serving as an alternative get or list which may use GET when possible. (See third bullet for specifics.)
```

Reply:
migrated to Post